features/step_definitions/admin/access_steps.rb

```diff
@@ -48,7 +48,7 @@
   go_to_admin_homepage
   expect(page).not_to have_text("Donations")
   visit admin_donations_path
-  expect(page).to have_current_path("/admin")
+  expect(page).to have_current_path("/admin?locale=en")
```

Comment:
@elliotcm This feels bad, but I've not been able to find a workaround. The fact that other checks, like the one on line 59 (`expect(page).to have_current_path(admin_donations_path)`), pass just fine makes me think that if we were to use a named route here there wouldn't be the requirement for `?locale=en`.

Thoughts?

Reply:
Gotcha.  You could try `admin_root_path`.

Reply:
Ah good shout. Let me try that now

Reply:
Works like a charm!